integration/setup/listener.go

```diff
@@ -183,11 +189,19 @@ func setupListener(tb testtb.TB, ctx context.Context, logger *zap.Logger, opts *
 			DisablePushdown:         *disablePushdownF,
 			CappedCleanupPercentage: opts.CappedCleanupPercentage,
 			CappedCleanupInterval:   opts.CappedCleanupInterval,
-			EnableNewAuth:           true,
+			EnableNewAuth:           opts.EnableNewAuth,
 			BatchSize:               *batchSizeF,
 			MaxBsonObjectSizeBytes:  opts.MaxBsonObjectSizeBytes,
 		},
 	}
+
+	if opts.EnableNewAuth {
```

Comment:
This was needed to test user management integration tests. Essentially, most tests run with new authentication like it is on `main`. Exception is `TestAuthenticationPLAIN` test that runs with old authentication.